src/ray/raylet/scheduling_queue.h

```diff
@@ -116,21 +116,35 @@ class SchedulingQueue {
   void MoveTasks(std::unordered_set<TaskID> tasks, TaskState src_state,
                  TaskState dst_state);
 
+  class TaskQueue {
+  public:
+    TaskQueue() {}
+    bool Append(const TaskID &task_id, const Task &task);
+    bool Remove(const TaskID &task_id);
+    bool Remove(const TaskID &task_id, std::vector<Task> &removed_tasks);
+
+    const std::list<Task> &GetTasks() const;
```

Comment:
nit: better add some simple documents for these functions.

Reply:
Accepted. thanks!